app/main.py

```diff
@@ -14,4 +14,10 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+
+    return counter
```

Comment:
the solution is correct but is it possible to make in 1 line without loop?

Reply:
I added new commit to this pull request

Reply:
@levseenko cool, but look at the last thread in the Q&A channel to see what to do next. You should request checking again